Payments/Integration/01_ManageConnection.md

```diff
@@ -0,0 +1,145 @@
+
+     The *Credentials* section is displayed below the drop-down list.
+
+    ![Credentials](../../Assets/Screenshots/Payments/Settings/CreateConnectionCredentials.png "[Credentials]")
+     > [Info] Depending on the selected driver, the fields in the *Credentials* section differ. 
+
+4. Enter the required data in the fields in the *Credentials* section. Use the information you received from the payment service provider to fill the credentials.   
+
+5. Click the [SAVE] button.   
+    The connection will be created. Depending on the driver, the *Checking credentials...* notice is displayed. After a few seconds, the *Loading data...* notice is displayed.
+
+     ![Checking credentials](../../Assets/Screenshots/Payments/Settings/CheckingCredentials.png "[Checking credentials]")
+
+    The data are synchronized between Actindo and the payment service provider, if a synchronization is necessary. The view for creating connections is automatically closed when the connection has been created and synchronized. The *Edit connection* view of the newly created connection is displayed. The *Settings* tab is selected.
+      ![Credentials](../../Assets/Screenshots/Payments/Settings/EditConnectionSettings.png "[Credentials]")
+    <!---Stefan: stimmt das? Ich kann es nicht nachmachen, wenn es Verbindungen sind, die nicht direkt aktiv sind, gibt es dann buttons mit denen man aktivieren, synchronisieren kann?-->
+
+
+
+
+## Edit PSP connection
+
+Edit a connection to adjust any changed credentials, the connection name or further settings. 
+
+#### Prerequisites
+
+At least one connection has been created, see [Create a connection](#create-a-psp-connection).
+
+#### Procedure
+
+*Payments > Settings > Tab CONNECTIONS*
+
+![Connections](../../Assets/Screenshots/Payments/Settings/Settings.png "[Connections]")
+
+1. Select the checkbox of the driver to select the connection you want to edit.  
+The editing toolbar is displayed above the list of connections.
+
+2. Click the ![Edit](../../Assets/Icons/Edit01.png "[Edit") (Edit) button.   
+    The *"Driver name" connection* view is displayed.
+
+    > [Info] Depending on the driver, the *Credentials* as well as the *Settings* tab are displayed in the view.
+
+    ![Credentials](../../Assets/Screenshots/Payments/Settings/EditConnectionCredentials.png "[Connection credentials]")
+
+2. Edit the desired data of the connection in the corresponding fields in the *Credentials* tab.   
+
+3. If necessary, click the *Settings* tab and edit the desired settings of the connection in the corresponding fields.
+
+4. Click the [SAVE] button.   
+    The *Checking credentials...* notice is displayed.
+    ![Checking credentials](../../Assets/Screenshots/Payments/Settings/CheckingCredentials.png "[Checking credentials]")
```

Comment:
Leerzeilen vor und nach dem Screenshot (das hat eine Auswirkung in GitHub, also die Leerzeilen fehlen auch auf der Webseite)

Reply:
done